UI/DropsController.gd

```diff
@@ -108,6 +122,36 @@ func _on_DropsTableItemsPanel_dropped_item(drop_item: GatheringItem):
 	_selected_drops_table.add_item(drop_item)
 	print_debug("Placed %s in drops table %s" % [tr(drop_item.item.name), _selected_drops_table.name])
 
+func _on_DropsFilterLineEdit_text_changed(new_text):
+	_FilterList(new_text)
+
+func _FilterList(filter_text: String = "") -> void:
+	var has_filter_text = !filter_text.empty()
+	
+	var filtered_tables: Array = []
+	var selected_table: DropsTable = _selected_drops_table
+
+	if selected_table != null and (selected_table.name.to_lower().find(filter_text.to_lower()) == -1):
+		filtered_tables.append(selected_table)
```

Comment:
If you're adding the selected table here and then below the selected table name matches the filter, itd be added twice to the list